### PR TITLE
Beast-mounted infantry record sheet improvements

### LINKED ISSUE #1358: Include name of the beast in the motive type for beast-mounted infantry record sheets
Currently, when exporting a record sheet for beast-mounted infantry, the motive type is just listed as the generic motive type for that beast, such as these two record sheets for the built-in Hipposaur and for a custom beast:
image
image

In fact, there's no explicit mention anywhere in this sheet that this infantry is beast-mounted. This is in contrast to official record sheets for beast-mounted infantry, such as those found in *Record Sheets: 3085 Unabridged - The Cutting Edge*.

On page 286 is Aerial Beast Infantry, where the motive type is given as *VOTL (Beast-Mounted, Branth)* [sic], and on page 321 is Beast Mounted Infantry, where the motive type is given as *Beast-Mounted, Tariq*. 

I just think it would be neat to include this information on the sheet, although it's probably not necessary.

### PR DESCRIPTION
**Pull request by pavelbraginskiy**
Adds the information mentioned in #1358 to the record sheets for beast-mounted infantry.

Some of examples of the generated record sheets:

image

image

Edit by Juliez:
Fixes #1358

### CODE REVIEW

File: megameklab/src/megameklab/printing/PrintInfantry.java [hunk 1]
```diff
@@ -99,62 +99,80 @@ protected void writeTextFields() {
...
+                // As of writing, the only time Mechanized SCUBA infantry might have nonzero walk mp is if it is beast-mounted.
+                if (infantry.getOriginalWalkMP() > 0) {
+                    mp2 = formatGroundMP();
+                    mode2 = "Ground";
+                }
                 break;
             case INF_MOTORIZED:
-                setTextField(MP_1, formatMovement(infantry.getWalkMP()));
-                setTextField(MODE_1, "Motorized");
+                mp1 = formatMovement(infantry.getWalkMP());
```
Comment (pavelbraginskiy) [inline megameklab/src/megameklab/printing/PrintInfantry.java:152]: Is there a reason that this isn't `formatGroundMp` btw? I have no idea so I just left it.

**Comment by pavelbraginskiy** [comment]
"Can operate in vacuum" feels weird but what do I know, maybe the Orcas have oxygen tanks.

**Comment by SJuliez** [comment]
Codewise all good. Hammer must approve RS changes.

**Comment by HammerGS** [comment]
RS changes are good as well. Thanks for the PR

**Review by SJuliez — APPROVED** [review]

**Pull request merged**